README.md

```diff
@@ -48,177 +83,849 @@ If you do not already have a **Developer Edition Account**, you can create one a
  
 ## Getting started
  
-Okta is available through [CocoaPods](http://cocoapods.org). To install
-it, simply add the following line to your Podfile:
+Okta is available through [CocoaPods](http://cocoapods.org). To install it, simply add the following line to your Podfile:
 
 ```ruby
-pod "OktaAuth"
+pod "OktaAuthSdk"
 ```
- 
-Construct a client instance by passing it your Okta domain name and API token:
- 
-[//]: # (method: createClient)
+
+## Usage guide
+
+SDK implements the following flows:
+- Primary authentication - allows you to verify username and password credentials for a user.
+- Multifactor authentication (MFA) - strengthens the security of password-based authentication by requiring additional verification of another factor such as a temporary one-time password or an SMS passcode. SDK supports user enrollment with MFA factors enabled by the administrator, as well as MFA challenges based on your Okta Sign-On Policy.
+- Unlock account - unlocks user account if it has been locked out due to excessive failed login attempts. **This functionality is subject to the security policy set by the administrator**.
```

Comment:
This is a good point to call out.  Are there some docs we could link to that discuss the policies that can be created?